### DISCUSSION TITLE
Migration Docs from 6.0.0 to 6.1.6

### DISCUSSION THREAD

**Discussion by lechu90ns** [serenity-is/Serenity#7033] [Help] [answered]
Hello All!

I am using docs to migrate into the 6.1.6 version but you link to the file that I do not have access to
https://github.com/serenity-premium/startsharp/blob/6a35bac28dccd4eb7720cf1094e330b2d6570aea/StartSharp/StartSharp.Web/Namespaces/tsconfig.json
https://github.com/serenity-premium/startsharp/blob/6a35bac28dccd4eb7720cf1094e330b2d6570aea/StartSharp/StartSharp.Web/Imports/tsconfig.json
https://github.com/serenity-premium/startsharp/blob/6a35bac28dccd4eb7720cf1094e330b2d6570aea/StartSharp/StartSharp.Web/tsconfig.json

Would you mind publicizing it? (edited)

**Comment by VictorTomaili** [comment, marked as answer]
Check this blog post: https://serenity.is/blog/2022/10/15/switching-to-es-modules
The file contents available in this post.
